guardrails/main.py

```
@@ -0,0 +1,118 @@
+                "message": {
+                    "role": "assistant",
+                    "content": response_content
+                },
+                "finish_reason": "stop"
+            }],
+            "usage": {
+                "prompt_tokens": 0,
+                "completion_tokens": 0,
+                "total_tokens": 0
+            }
+        }
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)